freqtrade/rpc/webhook.py

```diff
@@ -0,0 +1,66 @@
+"""
+This module manages webhook communication
+"""
+import logging
+from typing import Any,  Dict
+
+from requests import post, RequestException
+
+from freqtrade.rpc import RPC, RPCMessageType
+
+
+logger = logging.getLogger(__name__)
+
+logger.debug('Included module rpc.webhook ...')
+
+
+class Webhook(RPC):
+    """  This class handles all webhook communication """
+
+    def __init__(self, freqtrade) -> None:
+        """
+        Init the Webhook class, and init the super class RPC
+        :param freqtrade: Instance of a freqtrade bot
+        :return: None
+        """
+        super().__init__(freqtrade)
+
+        self._config = freqtrade.config
+
+    def cleanup(self) -> None:
+        """
+        Cleanup pending module resources.
+        This will do nothing for webhooks, they will simply not be called anymore
+        """
+        pass
```

Comment:
should you return `True` never the less?

Reply:
err no?
https://github.com/freqtrade/freqtrade/blob/266092a05d14e96631ef17a939aedcc5d93329a6/freqtrade/rpc/rpc.py#L64-L66